app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -34,6 +34,14 @@ def environment_url
       end
     end
 
+    def base_url
+      if BoltConfiguration.environments[environment].zero?
+        'https://connect.bolt.com'
+      else
+        'https://connect-sandbox.bolt.com'
+      end
```

Comment:
I know that bolt offers 3 envs: production, staging, and sandbox. Are you sure about that if?

Reply:
these are the only 2 options explicitly mentioned in the documentation: https://docs.bolt.com/developers/guides/checkout-guides/embedded-checkout/accounts/setup/#embedjs